conf/base.config

```diff
@@ -60,13 +60,13 @@ process {
         time   = { 24.h  * task.attempt }
     }
 
-    withName: 'FASTP' {
+    withName: 'FASTP*' {
         cpus = { 8 * task.attempt }
         memory = { 10.GB * task.attempt }
         time   = { 24.h  * task.attempt }
     }
 
-    withName: 'BWAMEM2_MEM' {
+    withName: 'BWAMEM2_MEM|MINIMAP2_ALIGN' {
```

Comment:
do they really require same resources?

Reply:
minimap2 actually uses about half the memory, so I can split these if you want

Reply:
done!